src/bojack/command.cr

```diff
@@ -2,27 +2,22 @@ require "./commands/*"
 
 module BoJack
   module Command
-    REGISTRY = [
-      BoJack::Commands::Get,
-      BoJack::Commands::Set,
-      BoJack::Commands::Delete,
-      BoJack::Commands::Size,
-      BoJack::Commands::Append,
-      BoJack::Commands::Pop,
-    ]
+    COMMANDS = {
+      "append" => BoJack::Commands::Append,
+      "delete" => BoJack::Commands::Delete,
+      "get"    => BoJack::Commands::Get,
+      "pop"    => BoJack::Commands::Pop,
+      "set"    => BoJack::Commands::Set,
+      "size"   => BoJack::Commands::Size,
+    }
 
     # Factory method for BoJack Commands
     #
     # It holds the logic to create the command instances
     #
     # @param command [String]
     def self.from(keyword) : BoJack::Commands::Command?
-      clazz = REGISTRY.find(nil) do |clazz|
-        clazz.keyword == keyword
-      end
-
-      return clazz.new if clazz
-
+      return COMMANDS[keyword].new if COMMANDS.has_key?(keyword)
```

Comment:
@mauricioabreu it really does not matter now, because this will be replaced in the near future, this method should always return a command or raise an exception. For now we still can't do that because we have missing commands...